README.md

```diff
@@ -157,6 +157,72 @@ which acts as a serial port. One such tool for accessing this serial port is
 [sercons](https://github.com/jclulow/vmware-sercons), though others (such as
 "screen") would also work.
 
+propolis-standalone also supports defining crucible-backed storage devices,
+thought it is somewhat inconvenient to do so without scripting. Currently you can only use a single region per block device. Read the comments in this TOML example for more details:
```

Comment:
probably worth linking to relevant crucible docs here in case any of documented values here change